tests/api/test_account.py

```diff
@@ -2764,7 +2764,7 @@ def test_query_customers_with_filter_placed_orders__(
         ({"field": "EMAIL", "direction": "ASC"}, ["John", "Leslie", "Joe"]),
         ({"field": "EMAIL", "direction": "DESC"}, ["Joe", "Leslie", "John"]),
         ({"field": "ORDER_COUNT", "direction": "ASC"}, ["John", "Leslie", "Joe"]),
-        ({"field": "ORDER_COUNT", "direction": "DESC"}, ["Joe", "John", "Leslie"]),
+        ({"field": "ORDER_COUNT", "direction": "DESC"}, ["Joe", "Leslie", "John"]),
```

Comment:
Why did you change order here?

Reply:
in sorters file I change enum values to list of fields use for sorting 
this way we don't have to worry about conflict when sorting by fields that aren't unique
as result some sorting have slightly different output
like in this example we sort by order_count and email